src/integration/pages/mediaAssetPage/ampTests.js

```diff
@@ -1,5 +1,7 @@
+import runCrossPlatformTests from './crossPlatformTests';
 import { runCoreAmpTests } from '../../common';
 
 export default () => {
+  runCrossPlatformTests();
   runCoreAmpTests();
 };
```

Comment:
I think this may be missing the AMP analytics tests too.

Reply:
Good spot